script/test

```diff
@@ -71,7 +71,7 @@ function runCoreRenderProcessTests (callback) {
   const cp = childProcess.spawn(executablePath, testArguments, {stdio: 'inherit', env: testEnv})
```

Comment:
1. This PR is sprawling enough with adding incidental rewrites to JavaScript to the mix!
2. This file, `script/test`, is already JavaScript anyway :wink: